Tests/SwiftLintFrameworkTests/VerticalWhitespaceRuleTests.swift

```diff
@@ -26,4 +28,24 @@ class VerticalWhitespaceRuleTests: XCTestCase {
         verifyRule(maxEmptyLinesDescription,
                    ruleConfiguration: ["max_empty_lines": 2])
     }
+
+    func testViolationMessageWithMaxEmptyLines() {
+        guard let config = makeConfig(["max_empty_lines": 2], ruleID) else {
+            XCTFail("Failed to create configuration")
+            return
+        }
+        let allViolations = violations("let aaaa = 0\n\n\n\nlet bbb = 2\n", config: config)
+        let verticalWhiteSpaceViolation = allViolations.first(where: { $0.ruleDescription.identifier == ruleID })
+        if let violation = verticalWhiteSpaceViolation {
```

Comment:
this test will silently fail if no violation is triggered. It should fail instead.